sycl/test-e2e/ESIMD/aot_mixed.cpp

```diff
@@ -5,10 +5,11 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: ocloc
-// RUN: %clangxx -fsycl -fsycl-targets=spir64_gen -Xsycl-target-backend=spir64_gen %gpu_aot_target_opts -o %t.sycl.out -DENABLE_SYCL=0 %s
+// TODO: Enable on other GPUs once internal ticket is fixed
+// REQUIRES: ocloc && gpu-intel-gen12
```

Comment:
FYI: I don't think we set `gpu-intel-gen12` in public CI anywhere, so it will likely disable the test completely there.

Reply:
@aelovikov-intel I'm trying to fix that in https://github.com/intel/llvm/pull/12627 :)